platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -1244,6 +1280,8 @@ public void start() {
         // a genesis event could be created here, but it isn't needed. This member will naturally create an
         // event after their first sync, where the first sync will involve sending no events.
 
+        final BasicConfig basicConfig = platformContext.getConfiguration().getConfigData(BasicConfig.class);
+
         final ParallelExecutor shadowgraphExecutor = PlatformConstructor.parallelExecutor(threadManager);
         shadowgraphExecutor.start();
         shadowgraphSynchronizer = new ShadowGraphSynchronizer(
```

Comment:
Would you refactor this variable name to `syncShadowgraphSynchronizer`? I keep getting confused as to it if is also used for chatter or not